src/app/work-item/work-item-board/work-item-board.component.html

```diff
@@ -1,20 +1,17 @@
 <div id="board_topWorkItems" class="container-fluid col-md-12 col-sm-12 col-xs-12 boardWrapper">
   <div class="board container-cards-pf">
-      <!-- board columns -->
-      <div class="board-lane" *ngFor='let lane of lanes'>
-
-        <section class="col-md-12 col-sm-12 col-xs-12 board-lane-column">
-          <div class="lane-header">
-            <span class="dib">{{lane.option}}</span>
-            <span class="issueCount dib" >{{workItems | almFilterBoardList:lane.option | count}}</span>
-          </div>
-          <div *ngFor='let item of workItems | almFilterBoardList:lane.option'>
+    <!-- board columns -->
+    <div class="board-lane" *ngFor='let lane of lanes'>
+      <section class="col-md-12 col-sm-12 col-xs-12 board-lane-column">
+        <div class="lane-header">
+          <span class="dib">{{lane.option}}</span>
```

Comment:
Yes, it's not needed. Because it is a span element and no need to make it inline-block until we need some top or bottom padding or margin.